Lib/test/test_clinic.py

```diff
@@ -2431,6 +2432,16 @@ class ClinicFunctionalTest(unittest.TestCase):
     locals().update((name, getattr(ac_tester, name))
                     for name in dir(ac_tester) if name.startswith('test_'))
 
+    def check_depr_star(self, pnames, fn, *args, **kwds):
+        regex = (
+            f"Passing( more than)?( [0-9]+)? positional argument(s)? to "
+            f"{fn.__name__}\(\) is deprecated. Parameter(s)? {pnames} will "
+            f"become( a)? keyword-only parameter(s)? in Python 3.14"
```

Comment:
```suggestion
            fr"Passing( more than)?( [0-9]+)? positional argument(s)? to "
            fr"{fn.__name__}\(\) is deprecated. Parameter(s)? {pnames} will "
            fr"become( a)? keyword-only parameter(s)? in Python 3.14"
```

Or the test crashes when it's run with `-We`:

```pytb
C:\Users\alexw\coding\cpython>python -We -m test test_clinic -v
Running Release|x64 interpreter...
== CPython 3.13.0a0 (heads/clinic/better-depr-tests:daea15eb23, Aug 9 2023, 11:32:33) [MSC v.1932 64 bit (AMD64)]
== Windows-11-10.0.22621-SP0 little-endian
== Python build: release with_assert
== cwd: C:\Users\alexw\coding\cpython\build\test_python_19188æ
== CPU count: 8
== encodings: locale=cp1252, FS=utf-8
0:00:00 Run tests sequentially
0:00:00 [1/1] test_clinic
test test_clinic crashed -- Traceback (most recent call last):
  File "C:\Users\alexw\coding\cpython\Lib\test\libregrtest\runtest.py", line 373, in _runtest_inner
    refleak = _runtest_inner2(ns, test_name)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\alexw\coding\cpython\Lib\test\libregrtest\runtest.py", line 313, in _runtest_inner2
    the_module = importlib.import_module(abstest)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\alexw\coding\cpython\Lib\importlib\__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1293, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1266, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1237, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 841, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 998, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1136, in get_code
  File "<frozen importlib._bootstrap_external>", line 1066, in source_to_code
  File "<frozen importlib._bootstrap>", line 400, in _call_with_frames_removed
  File "C:\Users\alexw\coding\cpython\Lib\test\test_clinic.py", line 2438
    f"{fn.__name__}\(\) is deprecated. Parameter(s)? {pnames} will "
                                                                   ^
SyntaxError: invalid escape sequence '\('

test_clinic failed (uncaught exception)

== Tests result: FAILURE ==
```

Reply:
Good catch